rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -270,6 +270,10 @@ def __init__(self): # pylint: disable=too-many-statements
         self.get_json_sample.restype = ctypes.c_int
         self.get_json_sample.argtypes = [ctypes.c_void_p, ctypes.c_char_p, ctypes.c_int, POINTER(c_char_p)]
 
+        self.get_json_instance = self.library.RTIDDSConnector_getJSONInstance
```

Comment:
Should we make a commit in develop that adds this function to ConnectorBinding.ifc / ConnectorBinding.c and returns a DDS_ReturnCode_t? I know that we cannot merge that into the release due to code freeze but at least it will not be forgotten in the future.

Reply:
yes